engine.py

```diff
@@ -7,12 +7,15 @@
 """
 import os
 import sys
-import ctypes
-import shutil
-import logging
 import traceback
 
+from PyQt4 import QtGui
+
+import sgtk
```

Comment:
Yeah I do, because when you are trying to call tank functions before calling the __init__ of the base class you don't have access to self.tank